packages/react-pages/src/node/cli.ts

```diff
@@ -17,34 +18,43 @@ console.log(
 )
 // console.log(chalk.cyan(`vite v${require('vite/package.json').version}`))
 
-// cli usage: vite-pages ssr [root] [vite config like --outDir or --configFile]
+// cli usage: vite-pages ssr [root] [--minifyHtml] [vite config like --outDir or --configFile]
 const [command, root] = argv._
 if (root) {
   argv.root = root
 }
 
 ;(async () => {
   if (!command || command === 'ssr') {
+    const toBeResovledConfig = {
+      configFile: argv.configFile,
+      build: {
+        outDir: argv.outDir,
+      },
+    }
+
     // user can pass in vite config like --outDir or --configFile
     const viteConfig = await resolveConfig(
-      argv,
```

Comment:
argv.root need to be passed to resolveConfig like this. Also we want to keep vite options like --base, --logLevel, --mode continue wo work.

Reply:
I just pushed a commit to improve it.

Reply:
The extra step is necessary, because `outDir` is located at [buildOptions](https://vitejs.dev/config/build-options.html#build-outdir),  rather than being placed directly under  `InlineConfig` .

Reply:
minimist can even parse this correctly, amazing😯